WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -30,7 +30,7 @@ + (NSArray *)routes
   @[
     [[FBRoute POST:@"/homescreen"].withoutSession respondWithTarget:self action:@selector(handleHomescreenCommand:)],
     [[FBRoute POST:@"/deactivateApp"] respondWithTarget:self action:@selector(handleDeactivateAppCommand:)],
-    [[FBRoute POST:@"/timeouts/implicit_wait"] respondWithTarget:self action:@selector(handleImplicitWaitCommand:)],
```

Comment:
Are you referring to this ? https://github.com/SeleniumHQ/selenium/commit/65dc17ed3435a0c4d1935e45314f4913f4e02d4e

This has already been refactored to be standard compliant on the java side. Not sure about the other clients (ruby, python, etc..)